<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="floorFrame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="floorNumComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>72</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Floor Number</string>
     </property>
    </widget>
    <widget class="QPushButton" name="floorUpButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="floorDownButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Floor </string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="elevatorFrame">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>531</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="carNumComboBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>72</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Car Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Request Floor</string>
     </property>
    </widget>
    <widget class="QComboBox" name="elevatorFloorRequestComboBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>72</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="currFloorDisplay">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(78, 154, 6);
background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="helpButton">
     <property name="geometry">
      <rect>
       <x>449</x>
       <y>100</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Floor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="speakButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>130</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Speak</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Elevator</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elevatorFireAlarmButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Fire Alarm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="embarkButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Embark (+80kg)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disembarkButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Disembark (-80kg)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openDoorButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closeDoorButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textDisplay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Text Display</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Audio Output</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="audioOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="initFrame">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="startSimulationButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Simulation</string>
     </property>
    </widget>
    <widget class="QLabel" name="numFloorsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Floors</string>
     </property>
    </widget>
    <widget class="QLabel" name="numElevatorsLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Elevators</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="numFloorsSpinBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>45</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="numElevatorsSpinBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>45</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="timeDependencyCheckBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Time Dependent</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="simControlFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="buildingFireButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Building Fire</string>
     </property>
    </widget>
    <widget class="QPushButton" name="powerOutageButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Power Outage</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetSimulationButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 41, 41);</string>
     </property>
     <property name="text">
      <string>Reset Simulation</string>
     </property>
    </widget>
   </widget>
   <zorder>simControlFrame</zorder>
   <zorder>initFrame</zorder>
   <zorder>floorFrame</zorder>
   <zorder>elevatorFrame</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
